buildSrc/src/main/groovy/airbyte-java-connector.gradle

```diff
@@ -53,6 +53,19 @@ class AirbyteJavaConnectorExtension {
                 }
             }
         } else {
+            def versionNumbers = cdkVersionRequired.split("\\.")
+            def majorVersion = Integer.valueOf(versionNumbers[0])
+            def minorVersion = Integer.valueOf(versionNumbers[1])
+            def hasKnownWarnings = false
+            if (majorVersion = 0 || majorVersion < 3) {
+                hasKnownWarnings = true;
```

Comment:
Can we do a regex match instead?
`def hasKnownWarnings = (cdkVersionRequired =~ "0\\.[0-2]\\.\\d+")` or whatever

Reply:
I tend to find regexp harder to read, but if you feel strongly about it, I'll do the change

Reply:
See comment below: I'd rather you maintain a hardcoded whitelist of connectors and check whether `project.name` is among them.